packages/opencensus-exporter-zpages/src/zpages-frontend/controllers/traceconfigz.controller.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2018 OpenCensus Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as tracing from '@opencensus/nodejs';
+import {classes} from '@opencensus/opencensus-core';
+import * as express from 'express';
+
+import {ZpagesExporter} from '../../zpages';
+
+import {TraceConfigzPageHandler} from './../page-handlers/traceconfigz.page-handler';
+
+exports.home = (req: express.Request, res: express.Response) => {
+  const html = new TraceConfigzPageHandler();
+  if (req.query) {
+    save(req.query);
+  }
+  res.send(html.emitHtml(req.query));
+};
+
+const save = (query) => {
```

Comment:
I made an improvement in the method name and added description to the methods. In relation to the type of the ```query``` it is not possible to determine because it comes from the ```express.Request``` class and there the query property type is **any**. I'll put any as query type here.

Reply:
I made an improvement in the method name and added description to the methods. In relation to the type of the query I created a new type with the expected query properties.